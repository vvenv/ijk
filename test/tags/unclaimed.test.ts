import { expect, test } from 'vitest';
import { parse } from '../__helper';

test('basic', () => {
  expect(parse('{{ > }}')).toMatchSnapshot();
  expect(parse('{{ < }}')).toMatchSnapshot();
  expect(parse('{{ alert("XSS") }}')).toMatchSnapshot();
  expect(parse('{{ window.alert("XSS") }}')).toMatchSnapshot();
});
