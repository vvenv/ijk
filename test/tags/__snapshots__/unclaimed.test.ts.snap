// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`basic 1`] = `""use strict";let s="";s+=e("{{ > }}");return s;"`;

exports[`basic 2`] = `""use strict";let s="";s+=e("{{ < }}");return s;"`;

exports[`basic 3`] = `""use strict";let s="";s+=e("{{ alert(\\"XSS\\") }}");return s;"`;

exports[`basic 4`] = `""use strict";let s="";s+=e("{{ window.alert(\\"XSS\\") }}");return s;"`;
